### PR TITLE
[TRUS-4162] - Add new page content for Agent - not authorised

### PR DESCRIPTION
**Pull request by jassalrichy**
Update AgentNotAuthorised page content & specs.
Remove TrustNotClaimed controller, view, specs, messages and end-point.

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -19,16 +19,30 @@ agentCannotAccessTrustYet.bullet.4 = If you need to close the trust, you will ne
...
-agentNotAuthorised.p2.a = Agent Services.
-agentNotAuthorised.p3 = The trustee who claimed the trust must use the same Government Gateway account to approve your request.
-agentNotAuthorised.p4 = If you need help with this service
-agentNotAuthorised.p4.a = contact the trusts helpline (opens in a new window or tab).
-agentNotAuthorised.p5 = You can
-agentNotAuthorised.p5.a = return to register and maintain a trust for a client.
+agentNotAuthorised.title = You cannot access this trust
+agentNotAuthorised.heading = You cannot access this trust
+agentNotAuthorised.p1 = To access the trust, one of the trustees needs to authorise your access so you can maintain the trust online on their behalf.
+agentNotAuthorised.subheading1 = The trustee needs to authorise your online access the trust
```
Comment (jackheslop96) [inline conf/messages.en:25]: The trustee needs to authorise your online access **to** the trust ?

File: conf/messages.en [hunk 2]
```diff
@@ -19,16 +19,30 @@ agentCannotAccessTrustYet.bullet.4 = If you need to close the trust, you will ne
...
+agentNotAuthorised.p2 = Through Agent Services you can create an authorisation link to send to the trustee. The trustee needs to use this link to authorise your access to the trust.
+agentNotAuthorised.p3 = The trustee needs to use the same Government Gateway account used to claim the trust to approve your request.
+agentNotAuthorised.subheading2 = What to do if your trustee cannot use the authorisation link
+agentNotAuthorised.p4 = If your trustee cannot complete the above steps, they need to
+agentNotAuthorised.p4.a = contact the trusts helpline (opens in a new window or tab)
+agentNotAuthorised.p5 = When the trustee contacts the helpline, they will be asked for:
+agentNotAuthorised.bullet.1 = information to verify their identity
+agentNotAuthorised.bullet.2 = the Unique Taxpayer Reference or Unique Reference Number of the trust
+agentNotAuthorised.p6 = The trustee will also be asked questions about their IT skills, and access to a computer. This is so HMRC can find out:
```
Comment (jackheslop96) [inline conf/messages.en:34]: Not keen on the content here - I'm not sure it makes sense. Suggestion for Scott:

The trustee will also be asked questions about their IT skills and whether they have access to a computer. This is so HMRC can find out:

File: conf/messages.en (hunk 1 above)
Comment (jassalrichy) [reply conf/messages.en]: Updated

File: conf/messages.en (hunk 2 above)
Comment (jassalrichy) [reply conf/messages.en]: Updated

**Pull request merged**